Suggest adding how to open migrated notebook (#3122)
MicrosoftDocs/OfficeDocs-SharePoint: migration/migrate-onenote-spmt.md

Folders are migrated to SharePoint as **OneNote Notebook** content rather than a normal folder with files. They'll appear in SharePoint like this:

![OneNote migration setting6](media/onenote-file-5.png)

> For this **OneNote Notebook** to appear in your Notebooks list in OneNote, you can click the **Show Actions** ellipses, next to the notebook name in the Document library, then select **Open > Open in app**, remember first, to close in the OneNote application the already migrated notebooks.

Reviewer: ```suggestion
For this **OneNote Notebook** to appear in your Notebooks list in OneNote, select the **Show Actions** ellipses next to the notebook name in the Document library, then select **Open > Open in app**. Remember to first close the OneNote application of the already migrated notebooks.
```
